docs/prometheus-integration.md

```diff
@@ -0,0 +1,103 @@
+# Prometheus Integration
```

Comment:
do we also add steps to configure (or one line mentioning which file to use).. for example
kubectl apply -f <path/to/antrea-prometheus.yml>
and any other step involved .. for those who simply want to go ahead and deploy it

Reply:
This config would most likely be a part of a larger Prometheus config which monitors other cluster elements.
But adding this to the doc won't do any harm :)